Add example to quote form to instruct format.

Murph33 opened the pull request.

seantaylor commented on app/assets/stylesheets/quotes.scss:
```diff
@@ -0,0 +1,3 @@
+.quote_example {
```
Our style guide uses dashes `-`, not underscores, `_`

seantaylor commented on app/views/quotes/_form.html.erb:
```diff
@@ -2,5 +2,11 @@
   <%= f.input :body %>
+  Please format as shown below:
+  <div>
+    <span class="quote_example">Quoted Person</span>
```
You probably could have put this class on the div and then had:

``` scss
.quote-example span {
  font-style: italic;
}
```

Since from this, it appears you want all spans in this block to have italic text. You may also be able to kill the `span` inside the `<q>` and say you want all `<q>` with italic font as well.

seantaylor commented on app/views/quotes/_quote.html.erb:
```diff
@@ -1,5 +1,5 @@
 <div>
-  <%= quote.quoted_person %> said, <q><%= quote.body %></q>, in the <%= quote.location %>
+  <%= quote.quoted_person %> said, <q><%= quote.body %></q>, in <%= quote.location %>.
```
This isn't part of adding an example quote. This is a _real_ quote :stuck_out_tongue:. Should be own commit.

Murph33 committed:
Change format of quote partial

Removed 'in the' from the partial since people can be quoted in
different states such as "while driving to work", "in the board room",
"on the phone", etc.  Provides more flexibility in the description of
how the quote was delivered.

Murph33 committed:
Add example to quote form to instruct format.

This provides an example of what the quote will be formatted like.
Think of Apples to Apples.

seantaylor commented on app/assets/stylesheets/quotes.scss:
```diff
@@ -0,0 +1,3 @@
+.quote-example span, q {
```
this likely doesn't do what you expect it to. This is equivalent to:

```
.quote-example span {
  font-style: italic;
}
q {
  font-style: italic;
}
```

You probably intended for: 

```
.quote-example span, .quote-example q {
  font-style: italic;
}
```

Or because we're writing sass, we can do:

```

.quote-example {
  span, q {
    font-style: italic;
  }
}
```

Which removes the need to duplicate the class.
